src/client/Client.cc

```diff
@@ -11244,13 +11244,21 @@ int Client::pwritev(int fd, const struct iovec *iov, int iovcnt, int64_t offset)
   return _preadv_pwritev(fd, iov, iovcnt, offset, true);
 }
 
+// NOTE: when called with zerocopy = true, the iovec SHOULD just consist of a
+// single entry with iov_len equal to the desired read length and iov_base
+// being a nullptr. The bufferlist pointer is used to return the data and the
+// caller is responsible for converting that into an iovec.
```

Comment:
I think better place for this comment is at line 11293

Reply:
I figured this was highlighting the interface to the function, I can move it if preferred.